tests/services/inventory/crawling_test.py

```diff
@@ -28,6 +28,7 @@ class NullProgresser(Progresser):
 
     def __init__(self):
         super(NullProgresser, self).__init__()
+        self.LOGGER = logger.get_logger(__name__)
```

Comment:
Can you please make the initialization of the `LOGGER` to be consistent with all the other files?  i.e. do it globally within the module, instead of class?

https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/google/cloud/forseti/services/explain/explainer.py#L19
```
LOGGER = logger.get_logger(__name__)
```